docs/blog/newsletter-september-2017.rst

```diff
@@ -0,0 +1,83 @@
+
+People also had some interesting perspectives on why the Stack Overflow (SO) effort failed:
+
+* No clear vision communicated for SO documentation
+* Return on investment for SO docs was too low to pursue
+* Docs are a hard sell--it's not easy to get people to volunteer to write or curate them
+* SO shut the Docs beta down before it had time to catch on
+* SO wasn't seen as the "right" place to go for docs (even if a company's official docs aren't well done)
+* No easy or effective way to address gaps in official docs in the SO environment (which is problem:solution-based)
+
+If you're interested in a lengthier dicsussion about the SO Documentation beta sunsetting and crowdsourced docs in general, check out last month's episode of the `Write The Docs podcast <http://podcast.writethedocs.org/2017/08/22/stack-overflow-failure-open-source-challenges/>`_!
+
+*************************************
+Doc Portfolios: A Perpetual Conundrum
+*************************************
+If you're job hunting right now, the likelihood is high that you've spent some time sprucing up your writing portfolio. But much more than, say, a journalist or a graphic designer, documentarians have to tread carefully when assembling our work samples. Much of the work we do is either so context-dependent that sharing our samples might not tell a prospective employer anything useful... or it's straight-up proprietary, in which case we can't share them at all. Maintaining a portfolio that's applicable, fresh, and accessible is a problem that, as one documentarian pointed out this month, is a "perpetual conundrum" we all face.
+
+The pitfalls of assembling and submitting a docs portfolio are varied. Content can be out of date or apply to outdated technologies. Depending on where the content was published and who by, it might no longer be available online. And if your work was done for internal audiences, it may well be that sharing it would be a breach of your contract. But even with all that, portfolios are often a mandatory part of the interview process.
+
+But just as the community was able to identify the problems with portfolio submissions, they were also able to brainstorm some solutions:
+* Ask your previous employers if they'd be comfortable with you sharing a small snippet of your work.
+* Edit all proprietary names and information out of your content before including it in your portfolio.
+* Consider contributing to an open source project and including those contributions in your portfolio.
+* If you're struggling to find online samples to link to ask if the organziation will accept a file submission instead of an online portfolio.
```

Comment:
add comma: "to link to, ask"

Reply:
what? I like it! Just gives the word a little je ne sais quoi...